cg/meta/workflow/taxprofiler.py

```diff
@@ -49,58 +47,47 @@ def __init__(
         self.nextflow_binary_path: str = config.taxprofiler.binary_path
         self.compute_env_base: str = config.taxprofiler.compute_env
 
-    def get_nextflow_config_content(self) -> str:
+    @property
+    def sample_sheet_headers(self) -> list[str]:
+        """Headers for sample sheet."""
+        return TaxprofilerSampleSheetEntry.headers()
+
+    @property
+    def _append_params_to_nextflow_config(self) -> bool:
+        """True if parameters should be added into the nextflow config file instead of the params file."""
+        return False
+
+    @property
+    def _multiple_samples_allowed(self) -> bool:
+        """Defines whether the analysis supports multiple samples to be linked to the case."""
+        return True
+
+    def get_nextflow_config_content(self, case_id: str) -> str:
         """Return nextflow config content."""
         return MULTIQC_NEXFLOW_CONFIG
 
     def get_bundle_filenames_path(self) -> Path:
```

Comment:
Could you define it as static here and also in the common api?

Reply:
this is not a staticmethod. The common one needs attributes, just for some of the children methods is not needed at the moment. I will leave this as it is

Reply:
I'm talking about `get_bundle_filenames_path`. It's static for rnafusion, but not for the rest implementations

Reply:
I commented the line 69, but I think github is trying to be smart here and include the code that you actually changed 😓

Reply:
oh sorry! I'll update that method then!